<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSCConfiguration</class>
 <widget class="QWidget" name="SSCConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="kcfg_AutoConnect">
     <property name="text">
      <string>Connect automatically on startup</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lbHost">
       <property name="text">
        <string>Host:</string>
       </property>
       <property name="buddy">
        <cstring>kcfg_Host</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KLineEdit" name="kcfg_Host"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbPort">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KIntSpinBox" name="kcfg_Port">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50000</number>
       </property>
       <property name="value">
        <number>4440</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="kcfg_Timeout"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Timeout:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_UseInstitutionSpecificIDs">
     <property name="text">
      <string>Use institution specific ids instead of ssc ids</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="lbInstitute">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Institute:</string>
       </property>
       <property name="buddy">
        <cstring>kcfg_Host</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KIntSpinBox" name="kcfg_ReferenceInstitute">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="lbInstitute_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Default language (ISO Code):
(requires restart)</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>kcfg_Host</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KLineEdit" name="kcfg_DefaultLanguage"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_UseInstitutionSpecificIDs</sender>
   <signal>toggled(bool)</signal>
   <receiver>lbInstitute</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_UseInstitutionSpecificIDs</sender>
   <signal>clicked(bool)</signal>
   <receiver>kcfg_ReferenceInstitute</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
